docs/platforms/javascript/common/distributed-tracing/index.mdx

```diff
@@ -29,6 +29,30 @@ What happens in the background is that Sentry uses reads and further propagates
 
 If you run any JavaScript applications in your distributed system, make sure that those two headers are added to your CORS allowlist and won't be blocked or stripped by your proxy servers, gateways, or firewalls.
 
+## Trace Duration
+
+<PlatformCategorySection supported={["browser"]}>
+
+In the browser, the SDK automatically starts a new trace in the following situation:
+
+- On page load: Whenever the page is (re-)loaded, a new trace is started. Simultaneously, a `pageload` span is created (see <PlatformLink to="/performance/instrumentation/automatic-instrumentation">Performance Monitoring</PlatformLink>). Once this span ends, the trace remains until the next navigation or page load. In case the server serving the initial page already started a trace and sent the necessary [HTML tags](./custom-instrumentation/#extract-tracing-information-from-html-meta-tags) to the browser, the SDK will continue this trace instead of starting a new one.
```

Comment:
```suggestion
- On page load: Whenever the page is (re-)loaded, a new trace is started. At the same time, a `pageload` span is created (see <PlatformLink to="/performance/instrumentation/automatic-instrumentation">Performance Monitoring</PlatformLink>). Once this span ends, the trace remains until the next navigation or page load. In case the server serving the initial page already started a trace and sent the necessary [HTML tags](./custom-instrumentation/#extract-tracing-information-from-html-meta-tags) to the browser, the SDK will continue this trace instead of starting a new one.
```